### ISSUE TITLE
Object property hasTopic has two ranges: "Topic" and "EditorialObject"

### ISSUE THREAD

**Issue by JuergenGrupp** [ebu/ebucoreplus#278]
The two ranges should be combined with an or-statement:

1) `EditorialObject` `hasTopic` (`Topic` or `EditorialObject`)

**Comment by tormodv** [comment]
It is not a bug, one programme might be the topic of another in the case you are making a behind-the-scenes production, but the typical use will be hasTopic -> Topic.

**Comment by tormodv** [comment]
Yes, this is a bug, but there might be more than one. "hasTopic" is a sub-property of hasSubject, but they do not have the same range. 
We have other properties without range, and the proper class restriction implicitly defines the range. 

<img width="155" alt="Skjermbilde 2023-08-13 kl  20 43 56" src="https://github.com/ebu/ebucoreplus/assets/12021922/8215df0e-3c5f-46dc-88a4-470831a7ebec">

So a solution can be to remove the hierarchy and the range statement.

**Comment by tormodv** [comment]
Or should we use "min 0" here too?

**Comment by tormodv** [comment]
What about EditorialObject concerning EditorialObject

or isAbout?

**Comment by JuergenGrupp** [comment]
I think we should use a new property like "isAbout", and not include it in the above mentioned hierarchy of hasSubject.
